net/datacenter-node-install/setup-cuda.sh

```diff
@@ -12,5 +12,6 @@ set -xe
 apt update
 apt install -y gcc make dkms
 
+# https://developer.nvidia.com/cuda-toolkit-archive
```

Comment:
This assumes that the runfiles are already located where the script is run from. Can `setup-partner-node.sh` make that assumption? Maybe check for it and download if not present?

Reply:
Auto-downloading is a little tricky because the NVIDIA archive URLs are inconsistent (version number in the path, but not filename.  `.run` extension may or may not be there. etc).  This is the main reason I punted it for now.

Reply:
But for any specific version it should be at most 1 `wget` and 1 `mv`, right?

Reply:
And/or an option for where to look for pre-downloaded ones, a mapping of URLs to filenames and plumbing it all through the top-level scripts.  We should definitely look into handling it automatically in the future.  I just have bigger fish to fry ATM

Reply:
Seems like you could skip the pre-download location option for an MVP?

Reply:
Ok, ok, you talked me into it :slightly_smiling_face:

Reply:
With apologies to my fellow Highland wifi users, https://github.com/solana-labs/solana/pull/5715/commits/36a8cf8a65fedd20f4b7c159dd76e074b7713949